config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -126,6 +128,39 @@ periodics:
       - --timeout=120m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190529-49b63a5-master
 
+- name: ci-kubernetes-e2e-windows-gce-k8sstable1
```

Comment:
we may want to consider using a `fork-per-release` annotation here in concert with [config-forker](https://github.com/kubernetes/test-infra/tree/master/experiment/config-forker)

Reply:
Is there an example that I can copy?